Composite/Composite/Transforms.fs

```diff
@@ -35,10 +35,44 @@ module Transforms =
                     while cur |> Option.isSome do
                         yield cur |> (function | Some x -> x | None -> failwith "Unexpected situation in enumeration.")
                         cur <- getNext ()

```

Comment:
Returning a current element to an output sequence may be done by taking a value of the option:
`cur.Value`

Reply:
@v-ilin Makes sense. In either case, the only possible situation here has `cur.Value` valid at all times.